spec/section/annotations.md

```diff
@@ -1,15 +1,154 @@
 ## Structural Annotations {#annotations}
 
-A [=field=] (`rml:field`) MUST contain zero or more [=structural annotations=]. 
+A [=field=] (`rml:field`) contains zero or more [=structural annotations=].
 <dfn>Structural annotations</dfn> give additional information about the [=field=] (`rml:field`) and its relation to the structure of its [=logical view=] (`rml:LogicalView`).
```

Comment:
@tirrolo Can you also update the shape and ontology files, and describe the general structure for a structural annotation in the spec?

Reply:
I began working on that this morning.